examples/nlp/glue_benchmark/glue_benchmark_config.yaml

```diff
@@ -0,0 +1,78 @@
+# GLUE Benchmark with pre-trained BERT models
+supported_tasks: &supported_tasks ['cola', 'sst-2', 'mrpc', 'sts-b', 'qqp', 'mnli', 'qnli', 'rte', 'wnli']
+
+trainer:
+  gpus: 1 # the number of gpus, 0 for CPU
+  num_nodes: 1
+  max_epochs: 3
+  max_steps: null # precedence over max_epochs
+  accumulate_grad_batches: 1 # accumulates grads every k batches
+  amp_level: O0 # O1/O2 for mixed precision
+  precision: 16
+  distributed_backend: ddp
+  checkpoint_callback: False  # Provided by exp_manager
+  logger: False  # Provided by exp_manager
+
+model:
+  task_name: &task_name mrpc # choose from: ["cola", "sst-2", "mrpc", "sts-b", "qqp", "mnli", "qnli", "rte", "wnli"] GLUE task name, MNLI includes both matched and mismatched tasks
+  supported_tasks: *supported_tasks
+  output_dir: null # dir to write write predictions
+  dataset:
+    data_dir: ??? # /path/to/data
+    max_seq_length: 128
+    use_cache: true
+
+    # shared across dataloaders:
+    num_workers:  2
+    pin_memory: false
+    drop_last: false
+
+  train_ds:
+    file_name: 'train.tsv'
+    shuffle: true
+    num_samples: -1
+    batch_size: 32
+
+  validation_ds:
+    file_name: 'dev.tsv' # for MNLI 'dev_matched.tsv' and 'dev_mismatched.tsv' will de used
```

Comment:
does the user have to change to 'dev_matched.tsv' and 'dev_mismatched.tsv' for MNLI?